docker/android.cmake

```diff
@@ -0,0 +1,33 @@
+# toolchain file for android targets, see #1110
+
+set(CMAKE_SYSTEM_NAME Android)
+set(CMAKE_SYSTEM_PROCESSOR "$ENV{CROSS_ANDROID_ARCH}")
```

Comment:
This is not the same as the arch elsewhere: it must be `i686`, `aarch64`, `x86_64`, `armv7-a`, or `armv5te`. Just setting `arm` will not work.